### DISCUSSION TITLE
Why is this repo called microsoft/microsoft-ui-xaml?

### DISCUSSION THREAD

**Discussion by allan2** [microsoft/microsoft-ui-xaml#7916] [General]
A discussion on a lighter note:

Why is this repo called **microsoft/microsoft-ui-xaml**?
Wouldn't **microsoft/ui-xaml** make more sense?

Or **microsoft/winui**? After all, the README title is "WinUI library"...

An initial study showed that it was 60% faster to type.
Further research is needed to test repo name alignment with user mental models ;)

**Comment by castorix** [comment]
Maybe because one of the main DLLs is named "Microsoft.ui.xaml.dll"...
I agree microsoft/WinUI would be better (but there is also microsoft/WindowsAppSDK)

**Reply by marcelwgn** [reply]
I think that's the main reason but maybe @jevansaks knows more here?
